web/src/components/AssertionForm/AssertionFormProvider.tsx

```diff
@@ -47,7 +42,6 @@ export const useAssertionForm = () => useContext<ICreateAssertionModalProviderCo
 
 const AssertionFormProvider: React.FC<{testId: string}> = ({children}) => {
   const dispatch = useAppDispatch();
-  const [drawerState, setDrawerState] = useState<DrawerState>(DrawerState.INITIAL);
```

Comment:
Removing the "drawer" (now it's called `BottomPanel`) state from the AssertionFormProvider.